scripts/standard-cpp-11.sh

```diff
@@ -22,7 +22,8 @@ change_readme() {
 #commit files and upload to github repo with new commmit sha
 echo "Committing changes, pushing to GH repo..."
 commit_and_upload() {
-  
+  git config --global user.email "devops@codecov.local"
```

Comment:
why `.local` here? instead of just `.io`?

Reply:
Made changes on newest commit